src/cpp/src/scheduler.hpp

```diff
@@ -376,4 +384,81 @@ class Scheduler {
         }
     }
 };
+
+    const std::string DEFAULT_POSTFIX = std::string();
+
+
+    /** Class to dump the current state of the KV block cache to disk as a number of text files, to be further parsed
+     * and visualized by the `cacheviz` tool.
+     */
+    class CacheStateDumper {
```

Comment:
could you please remove indentation within namespaces?

Reply:
Done, someone really should set up a clang-tidy config and run the formatting over the entire library since CLion puts these indents automatically and I couldn't find a setting to change that - we'll have these kinds of comments all the time if there isn't a clang-tidy in place.